### ISSUE TITLE
Add compatibility for Angel-125's new science system

### ISSUE THREAD

**Issue by CobaltWolf** [CobaltWolf/Bluedog-Design-Bureau#32]

**Comment by rasta013** [comment]
I'm going to be taking a look at this in the coming week or so and will chat with Angel about getting this pulled together.

**Comment by rasta013** [comment]
Ok.  Messaged Angel and he gave me some great details about how to get this setup.  Since I'm writing this from scratch it'll take me a bit to get it setup and tested but as soon as I've got a fully testable model I'll post here.  May have some questions about implementation along the way but by and large, this should be fairly easy to get done I think.

**Comment by CobaltWolf** [comment]
Dope!

**Comment by CobaltWolf** [comment]
@rasta013 are you going to be able to add this?

**Comment by CobaltWolf** [comment]
@jmrd98 would you be able to take care of this?

**Comment by jmrd98** [comment]
I ought to be able to take a look tomorrow after work. I'll look to see exactly how his stuff is rigged up. I know that the plugin is updated often enough that this should certainly be a 'supports' rather than 'bundles' sort of thing.

**Comment by CobaltWolf** [comment]
We try to depend on little, support as much as possible.

Matthew Mlodzienski
3D Animation and Visual Effects
Westphal COMAD 2017

**Comment by jmrd98** [comment]
This is still being poked at, but slowly. Work picked back up (work at a university --> semester starts) + I just bought a condo (Yay!), so things are a bit busy. I ran into a stumble with how mass is being set, or I (thought) I would have had something to post tonight. Unless you're cool with a 10-12 ton MOL, but that seems a bit overburdened :p

tldr: Still getting worked on, not entirely forgotten. If I can get the mass figured out, the system looks like a cool architecture.

**Comment by CobaltWolf** [comment]
alright, no real rush
